README.md

```diff
@@ -4,221 +4,627 @@
+
+  - **Type:** `str`
+  - **Requirement:** Required when neither the [`user`](#client-param-user) or [`password`](#client-param-password) parameters or the authentication environment variables are set.
+    See [User Authentication](#user-authentication).
+
+<a id="client-constructor-return-value"></a>
+#### Return Value
+
+Returns a new Frames `Client` data object.
+
+<a id="client-constructor-example"></a>
+#### Example
+
+The following example, for local platform execution, creates a Frames client for accessing data in the "users" container by using the authentication credentials of the user "iguazio":
+
+```python
+import v3io_frames as v3f
+client = v3f.Client("framesd:8081", user="iguazio", password="mypass", container="users")
+```
+
+<a id="client-common-method-params"></a>
+### Common Client Method Parameters
+
+All client methods receive the following common parameters:
+
+- <a id="client-method-param-backend"></a>**backend** &mdash; The backend data type for the operation.
+  See the backend-types descriptions in the [overview](#backend-types).
+
+  - **Type:** `str`
+  - **Valid Values:** `"csv"` | `"kv"` | `"stream"` | `"tsdb"`
+  - **Requirement:** Required
+
+- <a id="client-method-param-table"></a>**table** &mdash; The relative path to the backend data &mdash; A directory in the target platform data container (as configured for the client object) that represents a TSDB or NoSQL table or a data stream.
+  For example, `"mytable"` or `"examples/tsdb/my_metrics"`.
+
+  - **Type:** `str`
+  - **Requirement:** Required unless otherwise specified in the method-specific documentation
+
+Additional method-specific parameters are described for each method.
+
+<a id="method-create"></a>
+### create Method
+
+Creates a new TSDB table or a stream in a platform data container, according to the specified backend type.
+
+The `create` method is supported by the `tsdb` and `stream` backends, but not by the `kv` backend, because NoSQL tables in the platform don't need to be created prior to ingestion; when ingesting data into a table that doesn't exist, the table is automatically created.
+
+- [Syntax](#method-create-syntax)
+- [Common parameters](#method-create-common-params)
+- [`tsdb` backend `create` parameters](#method-create-params-tsdb)
+- [`stream` backend `create` parameters](#method-create-params-stream)
+
+<a id="method-create-syntax"></a>
+#### Syntax
+
+```python
+create(backend, table, attrs=None)
+```
+<!--
+create(backend, table, attrs=None, schema=None)
+-->
+<!-- [IntInfo] (26.9.19) The `schema` parameter is used only with the `csv`
+ test backend. -->
+
+<a id="method-create-common-params"></a>
+#### Common create Parameters
+
+All Frames backends that support the `create` method support the following common parameters:
+
+- <a id="method-create-param-attrs"></a>**attrs** &mdash; A dictionary of `<argument name>: <value>` pairs for passing additional backend-specific parameters (arguments).
+
+  - **Type:** dict
```

Comment:
The style of the parameter description here is different than the other methods (read,write..). is it on purpose?

Reply:
@talIguaz ultimately, all parameter descriptions should have separate type and requirement bullets (and valid and default values bullets where relevant), I just didn't have time to do it for this PR.